docs/user-guide/a1-01-matrix-layout.md

```diff
@@ -5,24 +5,48 @@ layout: user-guide
 Handling Matrix Layout Differences on Different Platforms
 ============================
 
-The differences on default matrix layout or storage conventions between GLSL (OpenGL/Vulkan) and HLSL has been an issue that frequently causes confusion among developers. When writing applications that work on different targets, one important goal that developers frequently seek is to make it possible to pass the same matrix generated by host code to the same shader code, regardless of what graphics API is being used (e.g. Vulkan, OpenGL or Direct3D). As a solution to shader cross-compilation, Slang provides necessary tools for developers navigate around the differences between GLSL and HLSL targets.
+The differences between default matrix layout or storage conventions between GLSL (OpenGL/Vulkan) and HLSL has been an issue that frequently causes confusion among developers. When writing applications that work on different targets, one important goal that developers frequently seek is to make it possible to pass the same matrix generated by host code to the same shader code, regardless of what graphics API is being used (e.g. Vulkan, OpenGL or Direct3D). As a solution to shader cross-compilation, Slang provides necessary tools for developers navigate around the differences between GLSL and HLSL targets.
+
+A high level summary:
+
+* Default matrix **layout** in memory for Slang is `column-major`. 
```

Comment:
I believe the default in Slang is `row-major`.

Reply:
On doing tests in #1814 it appears to be column-major? I can see an argument for making the default row-major.

Reply:
If that is the case, it is likely render-test is explicitly specifying column-major. We should double check if that is the case, since it contradicts with my experience when doing the examples.